respy/shared.py

```diff
@@ -26,37 +27,91 @@ def aggregate_keane_wolpin_utility(
     return value_function, flow_utility
 
 
-def create_base_draws(shape, seed):
-    """Create the relevant set of draws.
+def create_base_draws(shape, seed, monte_carlo_sequence):
+    """Create a set of draws from the standard normal distribution.
 
-    Handle special case of zero variances as this case is useful for testing.
-    The draws are from a standard normal distribution and transformed later in
-    the code.
+    The draws are either drawn randomly or from quasi-random low-discrepancy sequences,
+    i.e., Sobol or Halton.
+
+    `"random"` is used to draw random standard normal shocks for the Monte Carlo
+    exercises or because individuals face random shocks in the simulation.
+
+    `"halton"` or `"sobol"` can be used to change the sequence for two Monte Carlo
+    exercises. First, the calculation of the expected value function (EMAX) in the
+    solution and the choice probabilities in the maximum likelihood estimation.
+
+    For the solution and estimation it is necessary to have the same randomness in every
+    iteration or gradients do not exist. At the same time, the variance-covariance
```

Comment:
What do you mean by gradient does not exist? Are you referring to `chatter in the cirterion function`? See the book by Kenneth Train for details on this.
![chatter-criterion-function](https://user-images.githubusercontent.com/3607591/69448115-2f4bc780-0d58-11ea-8253-0e92673b451c.png)

Reply:
Yes, did not know how to describe it better. Added chatter in the text and referenced Train.